engine/src/test/java/io/camunda/zeebe/engine/util/TestStreams.java

```diff
@@ -109,8 +109,8 @@ public void withEventApplierFactory(
     this.eventApplierFactory = eventApplierFactory;
   }
 
-  public void withReplayMode(final ReplayMode replayMode) {
-    this.replayMode = replayMode;
+  public void withReplayMode(final StreamProcessorMode streamProcessorMode) {
```

Comment:
(same as above) 

Align the name of the method with the parameter name. We should change it to `streamProcessorMode()`.